packages/core/package.json

```diff
@@ -22,23 +22,16 @@
   ],
   "sideEffects": false,
   "exports": {
-    ".": {
-      "import": "./index.mjs",
-      "require": "./index.cjs"
-    },
+    ".": "./index.mjs",
```

Comment:
in a later release, would these eventually become `.js` (i.e. `index.js`)

seeing as we already have `"type": "module"`, they're already loaded as ES modules

Reply:
I prefer the `.mjs` for explicitness (in Node it might also slightly improve the performance as it skip one check to find package.json)